packages/commercetools/api-client/src/index.ts

```diff
@@ -53,7 +55,7 @@ const setup = <TCacheShape>(setupConfig: SetupConfig<TCacheShape>): ApolloClient
   locales = setupConfig.locales || locales;
   cookies = setupConfig.cookies || cookies;
   auth = setupConfig.auth || auth;
-  currentToken = setupConfig.currentToken || currentToken;
+  currentToken = setupConfig.forceToken ? setupConfig.currentToken : setupConfig.currentToken || currentToken;
```

Comment:
currentToken can be undefined in this case, so the type of this variable should be:
```TS
let currentToken: Token | undefined | null = null;
```
Or throw an exception when it happened